src/compiler/separate_compiler.nit

```diff
@@ -1942,6 +1995,47 @@ class SeparateRuntimeFunction
 		v.add("\}")
 		compiler.names[self.c_name] = "{mmethoddef.full_name} ({mmethoddef.location.file.filename}:{mmethoddef.location.line_start})"
 	end
+
+	fun compile_trampolines(compiler: SeparateCompiler)
```

Comment:
Even if it is quite self-explanatory, since it is public, can you document this ?